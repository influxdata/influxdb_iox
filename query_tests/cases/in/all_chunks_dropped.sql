-- Test for predicate push down explains
-- IOX_SETUP: OneMeasurementAllChunksDropped

-- list information schema
SELECT * from information_schema.tables;

-- same but shorter
SHOW TABLES;
