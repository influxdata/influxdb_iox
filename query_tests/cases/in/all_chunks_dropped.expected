-- Test Setup: OneMeasurementAllChunksDropped
-- SQL: SELECT * from information_schema.tables;
+---------------+--------------------+---------------+------------+
| table_catalog | table_schema       | table_name    | table_type |
+---------------+--------------------+---------------+------------+
| public        | iox                | h2o           | BASE TABLE |
| public        | system             | chunks        | BASE TABLE |
| public        | system             | columns       | BASE TABLE |
| public        | system             | chunk_columns | BASE TABLE |
| public        | system             | operations    | BASE TABLE |
| public        | information_schema | tables        | VIEW       |
| public        | information_schema | columns       | VIEW       |
+---------------+--------------------+---------------+------------+
-- SQL: SHOW TABLES;
+---------------+--------------------+---------------+------------+
| table_catalog | table_schema       | table_name    | table_type |
+---------------+--------------------+---------------+------------+
| public        | iox                | h2o           | BASE TABLE |
| public        | system             | chunks        | BASE TABLE |
| public        | system             | columns       | BASE TABLE |
| public        | system             | chunk_columns | BASE TABLE |
| public        | system             | operations    | BASE TABLE |
| public        | information_schema | tables        | VIEW       |
| public        | information_schema | columns       | VIEW       |
+---------------+--------------------+---------------+------------+
